device/coordinate_manager.cpp

```diff
@@ -455,6 +455,8 @@ std::shared_ptr<CoordinateManager> CoordinateManager::create_coordinate_manager(
                 tt::umd::blackhole::ARC_CORES,
                 tt::umd::blackhole::PCIE_GRID_SIZE,
                 tt::umd::blackhole::PCIE_CORES);
+        case tt::ARCH::Invalid:
```

Comment:
Ok, for now, but what about getting rid of this enum value altogether in the future?

Reply:
Makes sense but I see it is currently used at various places in metal. I should check if in one of those cases we don't want to throw, etc...